django/db/models/indexes.py

```diff
@@ -38,15 +40,15 @@ def check_name(self):
         return errors
 
     def create_sql(self, model, schema_editor):
-        fields = [model._meta.get_field(field) for field in self.fields]
+        fields = [model._meta.get_field(fo[0]) for fo in self.fields_orders]
```

Comment:
Rather than using tuple indexes, it's more readable to use tuple unpacking: `[model._meta.get_field(field_name) for field_name, order in self.fields_orders]`. I made that change when committing.